hazelcast/src/test/java/com/hazelcast/internal/jmx/ReplicatedMapMBeanTest.java

```diff
@@ -139,6 +145,35 @@ public void testAttributesAndOperations() throws Exception {
         assertEquals(0, size);
     }
 
+    @Test
+    public void testAttributeHitsAndOwnedEntryCountUpdatedAfterInterval()
+            throws Exception {
+        String firstKey = "firstKey";
+        String secondKey = "secondKey";
+        replicatedMap.put(firstKey, "firstValue");
+        replicatedMap.get(firstKey);
+        final String localHitsName = "localHits";
+        final String localOwnedEntryCountName = "localOwnedEntryCount";
+        long localHits = getLongAttribute(localHitsName);
+        long localEntryCount = getLongAttribute(localOwnedEntryCountName);
+        replicatedMap.get(firstKey);
+        replicatedMap.put(secondKey, "secondValue");
+        long localHitsNotUpdated = getLongAttribute(localHitsName);
+        long localEntryCountNotUpdated = getLongAttribute(localOwnedEntryCountName);
+        assertEquals(1, localHits);
+        assertEquals(1, localHitsNotUpdated);
+        assertEquals(1, localEntryCount);
+        assertEquals(1, localEntryCountNotUpdated);
+        assertTrueDelayed(1, new AssertTask() {
```

Comment:
`assertTrueDelayed` may wake up the thread slightly earlier than expected AFAIK. The test failed 4 out 100 times on my local machine. Also you might want to give it some extra milliseconds as actual update time and `lastUpdated` in `LocalStatsDelegate` may differ slightly.

You can use `sleepAtLeastMillis(int)` to ensure a certain amount of time passes.

Reply:
Done